src/withResolution/mediaQuery.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+type Callback = (mql: MediaQueryList, alias: string) => void
+type QueryData = {
+  [query: string]: {|
+    +mediaQuery: MediaQueryList,
+    +allCallbacks: (mql: MediaQueryListEvent) => void,
+  |},
+}
+type PerCallbackData = {
+  [query: string]: Array<{|
+    +alias: string,
+    +callback: Callback,
+  |}>,
+}
+
+const perCallbackData: PerCallbackData = {}
+const queryData: QueryData = {}
+
+function callCallbacks(query) {
```

Comment:
Maybe this could be called something like `getCallCallbacks` since this is a HOF whose returned function is the one that calls callbacks?

Reply:
What about `getCallbacksInvoker`?